### DISCUSSION TITLE
文对 "work hard，play hard"是反对意见吗?

### DISCUSSION THREAD

**Discussion by jiang-yuan** [zijie0/HumanSystemOptimization#11] [General]
工作日通过高强度的工作来获得成就和满足感 + 休息日进行各种休闲娱乐，运动，社交等方式来获得快乐
=释放多巴胺来获取快乐 + 释放多巴胺来获取快乐
=> 我们身体的多巴胺 baseline 会逐渐下降，出现一种耗尽（burn out）的心理感觉，对很多事物无法保持之前的兴趣与精力。

是这个逻辑吗?这样的话意思说不推荐work hard，play hard,但是没有提及一个替代方法,那怎么样算更好的方法呢?
而且高强度工作不一定能获得成就和满足感吧？和工作内容有关系。

**Comment by zijie0** [comment]
@jiang-yuan 好问题！我后续阅读了一本《不可打扰》，里面有一些相关的阐述对我挺有启发：
- 意志力可能并不是一种有限资源，而更像是情绪（会有起伏）。在控制情绪方面，自我暗示和调整的作用很强大。
- 自我控制本身就会带来不适。正如之前文中提到的很多内容，锻炼身体，控制饮食等等都是在跟很多基因驱使力作对。痛苦和不适很大程度上是人类进步的动力，一切需求都可以满足的乌托邦是没有未来的。

另外这本书里也针对专注力提出了很多有效的手段方法，对我这篇文章里的内容是个很好的补充。我目前的做法基本上把各种传统的休闲娱乐比如打游戏，看剧等压到最低，如无必要就不搞。当然陪伴家人，运动，社交这些还是保留的。

**Reply by jiang-yuan** [reply]
嗯,work hard是必须的,那降低多巴胺只能说平时的娱乐不要太疯来维持一下低baseline了,有点宗教苦修的意思了
OMEGA-3 大家买的一般都是强调DHA来加强专注力, 你写的EPA对情绪有调节作用,其实两个都挺有用的样子
